src/common/components/ui/formik/LabelField.tsx

```diff
@@ -32,6 +42,8 @@ export const LabelField: React.FC<LabelFieldProps> = ({
   onChange,
   validate,
   idPostfix,
+  forceUniqueKeys,
+  autocompleteValues = [],
```

Comment:
FIY the default value is always a new instance of array which will retrigger the `autocompleteRenderInput` useMemo on every rerender

Reply:
good point, fixed